lib/helpers/misc_helpers.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Helpers.MiscHelpers do
```

Comment:
This module can be `PhoenixSwagger.Helpers` 
Then at the usage sites, it can be aliased as: `alias PhoenixSwagger.Helpers`

Reply:
Fixed!